CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+## Katkıda bulunmadan önce
+
+Lütfen aşağıdaki seçeneklerin tamamı doğru yapılmışsa katkıda bulunun:
+
+- Katkı Çağrısı maddesini okudunuz mu?
+	Lütfen burayı okuyun: https://github.com/fatihacet/turkcekaynaklar-com/blob/master/Katki-Cagrisi.md
+
+- PR atmak istediğiniz branch ('compare:' yazan alan) master'dan farklı mı?
+	lütfen master branch üzerinden PR atmayın. Bu sağlıklı değildir. Fork reponuzda yeni bir branch oluşturup değişikliklerinizi orada yapın.
+
+- Fork reponuz güncel mi?
+	Lütfen en son değişiklikleri kendi reponuza güncelledikten sonra PR atın. Attığınız commit sayısından emin misiniz?
+
+- Çakışma var mı?
```

Comment:
Çakışma ifadesi yerine Conflict kullansak daha mi anlasilir olur? Sonucta GitHub'in Turkce destegi yok ve herkes ekranda Conflict gorecek, Çakışma degil.